sql-odbc/scripts/build_aws-sdk-cpp.ps1

```diff
@@ -23,13 +25,19 @@ Set-Location $BUILD_DIR
 # Configure and build 
 cmake $SRC_DIR `
     -A $WIN_ARCH `
+    -D CMAKE_VERBOSE_MAKEFILE=ON `
     -D CMAKE_INSTALL_PREFIX=$INSTALL_DIR `
     -D CMAKE_BUILD_TYPE=$CONFIGURATION `
     -D BUILD_ONLY="core" `
     -D ENABLE_UNITY_BUILD="ON" `
     -D CUSTOM_MEMORY_MANAGEMENT="OFF" `
     -D ENABLE_RTTI="OFF" `
-    -D ENABLE_TESTING="OFF"
+    -D ENABLE_TESTING="OFF" `
+    -D FORCE_CURL="ON" `
+    -D ENALBE_CURL_CLIENT="ON" `
```

Comment:
ENABLE is spelt wrong. Does this work?

Reply:
Fixed in e0672b3b. Hopefully, it affected nothing due to `FORCE_CURL`. Maybe it is odd.